test/unit/nedap/utils/test/impl.cljc

```diff
@@ -28,3 +27,27 @@
 
       {:nested {:key :value}}
       {:nested {:key :value}})))
+
+(def successful-summary {:fail 0, :error 0})
+
+(def unsuccessful-summary {:fail 1, :error 1})
+
+#?(:cljs
+   (deftest set-exit-code-for!
+     (are [summary existing-code expected] (testing [summary existing-code]
```

Comment:
`(testing [summary existing-code] ...)` haven't seen this usecase yet. Can see why it helps, as the reporting on _which_ are-line is broken is somewhat conflated. I guess it's as good as a text-approach 👍